enterprise/frontend/src/metabase-enterprise/comments/components/CommentEditor/editor.utils.ts

```diff
@@ -0,0 +1,42 @@
+import type { JSONContent } from "@tiptap/core";
+
+import type { DocumentContent } from "metabase-types/api";
+
+function isEmptyParagraphNode(node: JSONContent) {
+  if (node.type !== "paragraph") {
+    return false;
+  }
+  // Empty paragraph => no content or content has no text nodes with non-empty text
+  if (!node.content || node.content.length === 0) {
+    return true;
+  }
+  const hasNonEmptyText = node.content.some(
+    (child) =>
+      child.type === "text" &&
+      typeof child.text === "string" &&
+      child.text.trim().length > 0,
+  );
+  return !hasNonEmptyText;
+}
+
+export function trimTrailingEmptyParagraphsJSON(
```

Comment:
We don't need to do this. Not sure why would we only check for paragraphs either. 
Let's remove it.